src/rpc/ngHandlers/NFTInfo.cpp

```diff
@@ -31,7 +30,7 @@ namespace RPCng {
 NFTInfoHandler::Result
 NFTInfoHandler::process(NFTInfoHandler::Input input, Context const& ctx) const
 {
-    auto const tokenID = ripple::uint256{input.nftID.c_str()};
+    auto const tokenID = uint256{input.nftID.c_str()};
```

Comment:
@godexsoft these changes are to fix the bug i mentioned to you in your new implementation of `nft_info`. while i was here :)